cypress/integration/inventory/actions-menu.spec.js

```diff
@@ -1,23 +1,38 @@
 import TopMenu from '../../support/fragments/topMenu';
-import Actions from '../../support/fragments/inventory/actions';
+import InventoryActions from '../../support/fragments/inventory/inventoryActions';
+import InventorySearch from '../../support/fragments/inventory/inventorySearch';
 
 describe('ui-inventory: actions', () => {
-  beforeEach('navigates to Actions', () => {
+  beforeEach('navigates to actions', () => {
     cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
     cy.visit(TopMenu.inventoryPath);
   });
 
   it('C196752 verifies action menu options before any search is conducted', () => {
-    const optionsShouldDisabled = [
-      Actions.saveUUIDOption,
-      Actions.saveCQLQueryOption,
-      Actions.exportMARCOption,
-      Actions.showSelectedRecordsOption
-    ];
+    cy.do(InventoryActions.open());
 
-    Actions.open()
-      .then(() => {
-        optionsShouldDisabled.forEach(option => Actions.optionIsDisabled(option, true));
-      });
+    cy.expect(InventorySearch.getResults().absent());
+
+    InventoryActions.optionsIsDisabled([
+      InventoryActions.saveUUIDsOption(),
+      InventoryActions.saveCQLQueryOption(),
+      InventoryActions.exportMARCOption(),
+      InventoryActions.showSelectedRecordsOption()
+    ]);
+  });
+
+  it('C196753 verifies action menu options after searching and selecting result', () => {
+    cy.do([
+      InventorySearch.byEffectiveLocation([InventorySearch.effectiveLocation.mainLibrary]),
```

Comment:
I suggest to use InventorySearch.effectiveLocation.mainLibrary as default value. In this case you will have just InventorySearch.byEffectiveLocation(). And you can change this to extra wrapper with name  InventorySearch.getNotVoidResult() , which will have request of InventorySearch.byEffectiveLocation